cmd/admin-handlers.go

```diff
@@ -194,7 +194,7 @@ func (a adminAPIHandlers) ServiceActionHandler(w http.ResponseWriter, r *http.Re
 	case madmin.ServiceActionStop:
 		serviceSig = serviceStop
 	default:
-		logger.LogIf(ctx, fmt.Errorf("Unrecognized service action %s requested", action))
+		logger.LogIf(ctx, logger.AppLog, fmt.Errorf("Unrecognized service action %s requested", action))
```

Comment:
Log level IMO should come at the end here @poornas

Reply:
is there any particular reason why?

Reply:
The reason is all logging types either at in the beginning or at the end @poornas not in between.

Reply:
Here its like `logger.LogIf(context, message, kind)`